lib/src/flutter_soloud_bindings_ffi.dart

```diff
@@ -107,17 +107,28 @@ class FlutterSoLoudFfi {
       _lookup<ffi.NativeFunction<ffi.Void Function()>>('dispose');
   late final _dispose = _disposePtr.asFunction<void Function()>();
 
-  /// Load a new sound to be played once or multiple times later
-  ///
-  /// [completeFileName] the complete file path
-  /// [soundHash] return hash of the sound
-  /// Returns [PlayerErrors.noError] if success
-  ({PlayerErrors error, int soundHash}) loadFile(String completeFileName) {
+  /// Load a new sound to be played once or multiple times later.
+  ///
+  /// [completeFileName] the complete file path.
+  /// [loadIntoMem] if true Soloud::wav will be used which loads
+  /// all audio data into memory. This will be useful when
+  /// the audio is short, ie for game sounds, mainly used to prevent
+  /// gaps or lags when starting a sound (less CPU, more memory allocated).
+  /// If false, the audio data is loaded from the given file when
+  /// needed (more CPU less memory allocated).
+  /// See the [seek] note problem when using [loadIntoMem] = false
+  /// [soundHash] return hash of the sound.
+  /// Returns [PlayerErrors.noError] if success.
+  ({PlayerErrors error, int soundHash}) loadFile(
+    String completeFileName,
+    bool loadIntoMem,
```

Comment:
This is a small nit, but I'd prefer `loadIntoMemory`, at least in the developer-facing API.

Reply:
(See below for more options.)